testing/tests/ui/comparator-chaining.rs

```diff
@@ -0,0 +1,47 @@
+use askama::Template;
+
+#[derive(Template)]
+#[template(ext = "txt", source = "{{ a == b != c }}")]
+struct EqNe {
+    a: usize,
+    b: usize,
+    c: usize,
+}
+
+#[derive(Template)]
+#[template(ext = "txt", source = "{{ a <= b < c }}")]
+struct Between {
+    a: usize,
+    b: usize,
+    c: usize,
+}
+
+#[derive(Template)]
+#[template(ext = "txt", source = "{{ ((a == b) == c) == d == e }}")]
+struct ThreeTimesOk {
+    a: usize,
+    b: usize,
+    c: usize,
+    d: usize,
+    e: usize,
+}
+
+#[derive(Template)]
+#[template(ext = "txt", source = "{{ a == (b == (c == d == e)) }}")]
+struct ThreeTimesOk2 {
+    a: usize,
+    b: usize,
+    c: usize,
+    d: usize,
+    e: usize,
+}
+
+// Regression test for <https://github.com/askama-rs/askama/issues/454>
```

Comment:
Please also add what this is testing. :)

Reply:
I see no use in adding another comment besides the file name, the existing comment, and the expected outcome.

Reply:
Thanks! Always better to have context, makes it more reliable in case files are renamed, etc.